src/Microsoft.PowerShell.SHiPS/Resources/Resource.Designer.cs

```diff
@@ -10,8 +10,8 @@
 
 namespace Microsoft.PowerShell.SHiPS.Resources {
```

Comment:
This is auto-generated code. Can you please revert it?

Reply:
didn't get you, is it about reverting the whole change in Resource.Designer.cs file

Reply:
Yes please revert all changes made in this Resource.Designer.cs file.